operators/instana-agent-operator/2.0.5/manifests/instana-agent-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,452 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "instana.io/v1",
+          "kind": "InstanaAgent",
+          "metadata": {
+            "name": "instana-agent",
+            "namespace": "instana-agent"
+          },
+          "spec": {
+            "agent": {
+              "configuration_yaml": "# You can leave this empty, or use this to configure your instana agent.\n# See https://docs.instana.io/setup_and_manage/host_agent/on/kubernetes/\n",
+              "endpointHost": "ingress-red-saas.instana.io",
+              "endpointPort": "443",
+              "env": {},
+              "key": "replace-key"
+            },
+            "cluster": {
+              "name": "my-cluster"
+            },
+            "zone": {
+              "name": "edited-zone"
+            }
+          }
+        },
+        {
+          "apiVersion": "instana.io/v1beta1",
+          "kind": "InstanaAgent",
+          "metadata": {
+            "name": "instana-agent",
+            "namespace": "instana-agent"
+          },
+          "spec": {
+            "agent.endpoint.host": "ingress-red-saas.instana.io",
+            "agent.endpoint.port": 443,
+            "agent.env": {
+              "INSTANA_AGENT_TAGS": "example"
+            },
+            "agent.key": "replace-me",
+            "agent.zone.name": "my-zone",
+            "cluster.name": "replace-me",
+            "config.files": {
+              "configuration.yaml": "# You can leave this empty, or use this to configure your instana agent.\n# See https://docs.instana.io/setup_and_manage/host_agent/on/kubernetes/\n"
+            }
+          }
+        }
+      ]
+    capabilities: Full Lifecycle
+    categories: Monitoring,Logging & Tracing,OpenShift Optional
+    certified: "false"
+    containerImage: instana/instana-agent-operator@sha256:dfd04450a62008fa4cee15d7819d1ff8ea4deecdfadf4a570d414fd939276551
```

Comment:
Please prepend host like `docker.io or quay.io`

Reply:
Thanks for review and I updated the PR with a valid image path